Author: Diane Trout <diane@ghic.org>
Description: Allow disabling SSL errors in rakia.
Origin: https://git.reviewboard.kde.org/r/121877/
--- a/plugins/rakia/rakia-advanced-options-widget.cpp
+++ b/plugins/rakia/rakia-advanced-options-widget.cpp
@@ -47,6 +47,8 @@ RakiaAdvancedOptionsWidget::RakiaAdvancedOptionsWidget(ParameterEditModel* model
     handleParameter(QLatin1String("transport"),  QVariant::String,
                     m_ui->transportComboBox,
                     m_ui->transportLabel);
+    handleParameter(QLatin1String("ignore-tls-errors"), QVariant::Bool,
+                    m_ui->ignoreTlsErrorsCheckBox, 0);
 
     handleParameter(QLatin1String("stun-server"),        QVariant::String,
                     m_ui->stunServerLineEdit,
diff --git a/plugins/rakia/rakia-advanced-options-widget.ui b/plugins/rakia/rakia-advanced-options-widget.ui
index 460d839..953419a 100644
--- a/plugins/rakia/rakia-advanced-options-widget.ui
+++ b/plugins/rakia/rakia-advanced-options-widget.ui
@@ -115,6 +115,13 @@
         </property>
        </widget>
       </item>
+      <item row="4" column="1">
+       <widget class="QCheckBox" name="ignoreTlsErrorsCheckBox">
+        <property name="text">
+         <string>Ignore TLS Errors</string>
+        </property>
+       </widget>
+      </item>
      </layout>
     </widget>
    </item>
